resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/ContextInjector.java

```diff
@@ -0,0 +1,9 @@
+package org.jboss.resteasy.spi;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+public interface ContextInjector<WrappedType, UnwrappedType> {
+   // FIXME: remove rawType and genericType?
```

Comment:
What is the plan about this?

Reply:
I guess I'd remove the types because I don't see their point, but I based this on `ContextResolver` from JAX-RS which has `Class` (and then added `Type`). @asoldano any idea why they have that `Class` param ? I couldn't guess any use for it, since an injector can only declare injection of one type, since it's duplicated in the class signature via the type parameter.